Perl/Perl.sublime-syntax

```diff
@@ -945,7 +945,7 @@ contexts:
 
   sub:
     - match: \bsub\b
-      scope: keyword.control.sub.perl
+      scope: keyword.declaration.sub.perl
```

Comment:
It it's comparable to a function, shouldn't it be scoped `keyword.declaration.function` in the general sense?